doc/mssql_examples_test.go

```diff
@@ -0,0 +1,61 @@
+package main
```

Comment:
Only documentation (md files) should be in the doc folder. If this example test works as a standalone test, please put it in the root folder.

Since this example is targeting the `SetQueryNotification`, you should rename is to something like `setquerynotification_example_text.go`.

If you look at other *_examples_test.go files (e.g., `datetimeoffset_example_test.go`), you will see that instead of identifying is as `package main`, they use `package mssql_test`. That way if this file is in the root folder along with the src files, you will not get package clash (having 2 packages in the same directory). When godoc gets this example, they'll replace `package mssql_test` with `package main` so it looks like a proper test application.

Reply:
Since the function has "Example" prefixing it's name it won't be run as a test. I have moved it from the doc folder though.